src/main/java/com/cburch/logisim/data/TestVector.java

```diff
@@ -131,7 +131,11 @@ private void parseHeader() throws IOException {
             throw new IOException("Test Vector header format error: bad spec: " + t);
 
           columnName[i] = t.substring(0, s);
-          int w = new Integer(t.substring(s + 1, e)).intValue();
+          int w = 0;
+          try {
+            w = Integer.parseInt(t.substring(s + 1, e));
+          } catch (NumberFormatException ex) {
+          }
 
           if (w < 1 || w > 32)
```

Comment:
@BFH-ktt1: Isn't the maximum bit width now 64 bits? May be, there are still other places assuming 32 bits.

Reply:
@maehne , correct, however I do not know how up-to-date the test-vector code is in respect to the 64-bit width. I have to look into the details, on my todo list.

Reply:
I tried changing the `32` here to `64`. The Test Vector file header is parsed correctly, but I then got a different error message, because the 64 bit values are not parsed correctly from the Test Vector file.

I created #507 for those problems.